src/psyclone/tests/test_files/gocean1p0/kernel_scalar_float.f90

```diff
@@ -14,9 +14,10 @@ module kernel_scalar_float
   !=======================================
 
   type, extends(kernel_type) :: bc_ssh
-     type(go_arg), dimension(3) :: meta_args =              &
+     type(go_arg), dimension(4) :: meta_args =                 &
           (/ go_arg(GO_READ,      GO_R_SCALAR, GO_POINTWISE),  &
              go_arg(GO_READWRITE, GO_CT,       GO_POINTWISE),  &
+             go_arg(GO_READ,      GO_GRID_X_MAX_INDEX),        &
```

Comment:
Ah! So you've extended the test to exercise something that was previously broken?

Reply:
Yep :+1: